dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/SVarCStandard.mo

```diff
@@ -72,7 +72,7 @@ protected
   final parameter Types.PerUnit BVar0Pu = B0Pu - BShuntPu "Start value of variable susceptance in p.u";
   parameter BaseControls.Mode Mode0 "Start value for mode";
   parameter Boolean selectModeAuto0 = true "Start value of the boolean indicating whether the SVarC is initially in automatic configuration";
-  parameter Integer setModeManual0 = 2 "Start value of the mode when in manual configuration";
+  final parameter Integer setModeManual0 = Integer(Mode0) "Start value of the mode when in manual configuration";
```

Comment:
Should'nt it be changed also for SVarCPV.mo ?

Reply:
You're right!! Done. Sorry I forgot to commit the corrections I made at last when I noticed there was a problem with SVarCPV. Thanks!